RayLin.md

```diff
@@ -32,5 +41,5 @@
 
 ### Autofill Planning ###
 
-  - [November ~]
-    - Discuss/Work together with Steve on two preference dialogs.
+  - [December ~]
+    - join autofill sessions in Hawaii all hands
```

Comment:
If you are optimistic, maybe you should spend time with Luke/Steve/Matt most of the time in Hawaii. We want to figure out a plan which most of the bugs are unblocked and ready to be worked on, for incoming engineers like you (and maybe me too... I am going to write some patches myself). So it's better if you know these plans.

It would also help if you join the #formfill IRC channel.